.github/workflows/cbbr_build.yml

```diff
@@ -15,10 +15,22 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./db-cbbr/cbbr_build
+    container:
+      image: nycplanning/build-geosupport:latest
+    env:
+      BUILD_ENGINE: postgresql://postgres:postgres@postgis:5432/postgres
+      EDM_DATA: ${{ secrets.EDM_DATA }}
+      AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
     services:
-      postgres:
-        image: postgis/postgis:12-3.0-alpine
+      postgis:
+        image: postgis/postgis:14-3.3-alpine
```

Comment:
no downstream changes needed when this service's name changes?

Reply:
The build_engine reflects this change

Reply:
oops sorry didn't see the text change within the line change